core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -263,7 +263,12 @@ class CivilizationInfo {
                 } +
                 policies.policyUniques.getUniques(uniqueTemplate) +
                 tech.techUniques.getUniques(uniqueTemplate) +
-                temporaryUniques.filter { it.first.placeholderText == uniqueTemplate }.map { it.first }
+                temporaryUniques.filter { it.first.placeholderText == uniqueTemplate }.map { it.first } +
+                if (religionManager.religion != null) 
+                    religionManager.religion!!.getFounderUniques()
```

Comment:
. asSequence should be before .filter

Reply:
Maybe. With expected average cardinality near or below 1 _could_ be marginally more efficient the other way round.

What I see here is repeated lines ending in an operator. Coding style and kotlin not needing `;` ... You know, if you _want_ the operators _leading_ the line for readability, that can be done - just have an unmatched bracket open... I think. Note to self.